### DISCUSSION TITLE
群晖7.0下docker+cloudsync上传时为何总是出现以下类似错误

### DISCUSSION THREAD

**Discussion by huamike** [messense/aliyundrive-webdav#533] [Q&A] [answered]
docker运行日志如下：
vfs  upload file chunk 252 failed   
  <Code>AccessDenied</Code>
 <Message>Request has expired.</Message>
  <RequestId>62E1E0140BC25A383008D868</RequestId>

实际运行体验是，十来个大文件理论上只要3-4小时可以传完，结果传了一晚上只上传了3个

**Comment by messense** [comment, marked as answer]
上传的 url 过期了，这个可以调整下。

https://github.com/messense/aliyundrive-webdav/commit/6ae72aecd121d96958571415434f5b02e80d6f94
